### PR TITLE
Moved 'TIP' on strings to end in string_ids.h (standard) + Aligned new descriptions up in cheats.c (Done)

### PR DESCRIPTION
**Pull request by Nubbie**
1.. Changed all the string tips in `src/localisation/string_ids.h` to include the `_TIP` at the end as a standard for tooltip strings, as Gymnasiast told me in #3403   
_(Obs. To find all these tips with _TIP_ in the middle, I searched for `TIP_`, _all of the strings was just cheats)_

> We always put TIP at the end

2.. Changed the strings to match + added spacings between the lines to make everything line up and look cleaner inside `src/windows/cheats.c` _(Should have been in separate commits..)_

_All the strings are there, you can ctrl+f to see that they are used on both places_

### CODE REVIEW

**Comment by Nubbie** [comment]
Guess the tests didn't like that.. might have to remove all the 'tabs'

File: src/windows/cheats.c [hunk 1]
```diff
@@ -146,108 +146,108 @@ enum WINDOW_CHEATS_WIDGET_IDX {
...
-	{ WWT_CLOSEBOX,			1,		XPL(1),					WPL(1),					YPL(2),			HPL(2),			STR_CHEAT_10_MINUTE_INSPECTIONS,	STR_CHEAT_10_MINUTE_INSPECTIONS_TIP },	// 10 minute inspections
-	{ WWT_CHECKBOX,			2,		XPL(0),					OWPL,					YPL(7),			OHPL(7),		STR_CHEAT_SHOW_ALL_OPERATING_MODES,	STR_NONE }, 							// Show all operating modes
-	{ WWT_CHECKBOX,			2,		XPL(0),					OWPL,					YPL(6),			OHPL(6),		STR_CHEAT_SHOW_VEHICLES_FROM_OTHER_TRACK_TYPES,	STR_NONE }, 				// Show vehicles from other track types
+	{ WWT_CLOSEBOX,			1,		XPL(0),				WPL(0),				YPL(0),			HPL(0),			STR_CHEAT_RENEW_RIDES,					STR_CHEAT_RENEW_RIDES_TIP },			// Renew rides
+	{ WWT_CLOSEBOX,			1,		XPL(1),				WPL(1),				YPL(1),			HPL(1),			STR_CHEAT_MAKE_DESTRUCTABLE,				STR_CHEAT_MAKE_DESTRUCTABLE_TIP },		// All destructable
+	{ WWT_CLOSEBOX,			1,		XPL(0),				WPL(0),				YPL(1),			HPL(1),			STR_CHEAT_FIX_ALL_RIDES,				STR_CHEAT_FIX_ALL_RIDES_TIP },			// Fix all rides
+	{ WWT_CHECKBOX,			2,		XPL(0),				OWPL,				YPL(11),		OHPL(11) + 8,		STR_CHEAT_UNLOCK_OPERATING_LIMITS,			STR_CHEAT_UNLOCK_OPERATING_LIMITS_TIP }		// 410 km/h lift hill etc.
```
Comment (HaasJona) [inline src/windows/cheats.c]: You removed a comma here, it seems.

Comment (Nubbie) [inline src/windows/cheats.c]: Thanks for telling me! :D :+1:

**Comment by Gymnasiast** [comment]
Please revert the whitespace 'fix'. We use a tab width of 4 spaces. By default, Github assumes it is 8 and therefore shows it incorrectly. You can set the tab space to 4 in the online editor.

**Comment by Nubbie** [comment]
In other words - Remove all the 'whitespace fixes'?

**Comment by Nubbie** [comment]
It's now done.

In the end: All the strings that had _tip_ in the middle was cheats, so it's now all been moved to the back. The other thing is that the descriptions I added before have gotten their explanations aligned with the others (with the correct width 4 this time, thanks Gymnasiast!)

**Comment by Gymnasiast** [comment]
:+1: from me.

**Pull request merged**